docs/src/modules/concepts/pages/architecture-model.adoc

```diff
@@ -1,24 +1,46 @@
-= Service structure and layers
+= Project structure
 
 include::ROOT:partial$include.adoc[]
 
-Akka allows developers to focus on domain modeling and Application Programming Interfaces (APIs) without needing to manage low-level persistence or concurrency concerns. The organization of an Akka service into clear layers makes this possible.
+Akka encourages a project structure that separates your system’s Application Programming Interfaces (APIs), Akka component logic, and business logic into different directories.
 
-== Layered structure
+This structure supports a clear separation of concerns. It helps enable iterative development, testing in isolation, predictable packaging, and the ability to externalize configuration and static assets.
 
-Akka services follow a layered model where each layer has a distinct role. The layers are arranged concentrically, like an onion. The innermost layer holds business logic. Outer layers coordinate and expose that logic to the outside world.
+== Akka project structure
 
-image::docs-onion_architecture-v1.min.svg[Layered Structure of a Service, width=400]
+A typical Akka project might have a layout like the following:
 
-Each layer should live in its own package:
+[source,txt]
+----
+src/
+ ├── main/
+ │   ├── java/acme/planningagent/
+ │   │   ├── api/           # External MCP, HTTP, gRPC endpoints
+ │   │   ├── application/   # Akka components: Agents, Entities, Views, Consumers, Timers, Workflows
+ │   │   │   └── tool/      # MCP or functional tools
```

Comment:
Is there a reason to highlight `tools` that much? I would just say that it's a part of the application. How the application package is organized is up to the user.

Reply:
https://github.com/akka/akka-sdk/pull/759#discussion_r2185240777